### ISSUE TITLE
RadCalendar: Date spanning multiple days is only displayed on start date

### ISSUE THREAD

**Issue by neddinn** [telerik/nativescript-ui-feedback#694]
> Please take a minute to read our [NativeScript Code of Conduct](https://github.com/NativeScript/codeofconduct) before proceeding with posting issues or discussing. The purpose of this guide is to make communication and cooperation within our forums a pleasure for you and the other members.

### Did you verify this is a real problem by searching the [NativeScript Forum](https://discourse.nativescript.org/)?
Yes

### Tell us about the problem
Creating an event that spans multiple days only shows on the start day in the view? I'm doing something like this to create the event
```
  var now = new Date();
  var today = new Date(now.getFullYear(), now.getMonth(), 6);
  var tomorrow = new Date(now.getFullYear(), now.getMonth(), 7);
  var event1 = new calendarModule.CalendarEvent('first', today, tomorrow, false);
  dataSource.set('events', [event1]);
```
But it only shows on the 6th in the view instead of 6th and 7th?

### Which platform(s) does your issue occur on?
iOS

### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI version:** ("nativescript-pro-ui": "^3.2.0")
- **CLI:** (4.0.0)
- **Cross-platform modules:** (3.3.0)
- **Runtime(s):** (tns-ios: 3.3.0-rc-2017-10-24-1)

### Please tell us how to recreate the issue in as much detail as possible.
1. Start the application ..
2. Add the radcalendar module
3. Populate with data spanning multiple days

**Comment by tsonevn** [comment]
The component provided by NativeScript UI were separated into standalone plugins. You can review our migration steps [here](https://docs.telerik.com/devtools/nativescript-ui/migration) and check if you will be able to recreate the issue with the latest `nativescript-ui-calendar` version. If the problem persists, please send us sample project, which can be used for debugging.

**Comment by neddinn** [comment]
Hi @tsonevn ,
Its the same issue with `nativescript-ui-calendar`. Here is a sample project https://github.com/neddinn/calendar-test/

**Comment by neddinn** [comment]
Hi guys, any update on this issue?

**Comment by tsonevn** [comment]
Hi @neddinn,
The problem is still under review. At this time I can suggest keeping track of the issue, and we will provide more info on it.

**Comment by ThrowingDarts** [comment]
This is a severe issue making the calendar useless if spanning multiple dates. If you can't fix this issue iOS is there a work-a-round? Is there any idea when this will be fixed?

**Comment by tgpetrov** [comment]
Hi @ThrowingDarts,
We can't commit to a specific date for the fix. For now, the workaround would be instead of adding one event (from 5pm today to 5pm tomorrow) to add two events (one from 5pm to 12am today and another from 12am to 5pm tomorrow). Keep track on the issue for further updates.

**Comment by zbranzov** [comment]
This issue should be fixed in nativescript-ui-calendar@3.8.1
